Dockerfile

```diff
@@ -22,17 +22,18 @@ ARG RUN_IMAGE=openmicroscopy/omero-${COMPONENT}:latest
 
 
 FROM ${BUILD_IMAGE} as build
+USER root
 RUN apt-get update \
  && apt-get install -y ant \
       python-pip python-tables python-virtualenv python-yaml python-jinja2 \
       zlib1g-dev python-pillow python-numpy python-sphinx \
       libssl-dev libbz2-dev libmcpp-dev libdb++-dev libdb-dev \
       zeroc-ice-all-dev \
- && pip install --upgrade 'pip<10' setuptools
+ && pip install --upgrade 'pip<10' setuptools flake8==2.4.0 pytest==2.7.3
 # TODO: unpin pip when possible
 # openjdk:8 is "stretch" or Debian 9
 RUN pip install https://github.com/ome/zeroc-ice-py-debian9/releases/download/0.1.0/zeroc_ice-3.6.4-cp27-cp27mu-linux_x86_64.whl
-RUN adduser omero
+RUN id 1000 || useradd -u 1000 -ms /bin/bash build
```

Comment:
Nice way to conditionally add the user and enable the base image to be overriden at build time.